tst/ProducerFunctionalityTest.cpp

```diff
@@ -61,7 +61,7 @@ TEST_F(ProducerFunctionalityTest, reset_stream_on_stream_error_before_putFrame)
 {
     STREAM_HANDLE streamHandle = INVALID_STREAM_HANDLE_VALUE;
     UINT32 i;
-    UINT32 totalFragments = 60;
+    UINT32 totalFragments = 10;
```

Comment:
Make sure that these do what's intended - some would need enough fragments to roll over to the next streaming session

Reply:
yes, the ones that I changed dont require to reach token rotation.